live-examples/html-examples/interactive-elements/css/menu.css

```diff
@@ -1,5 +1,5 @@
 .news {
-  background-color: bisque;
+  background-color: black;
```

Comment:
this makes it more accessible in dark mode, but makes the link above impossible to read in light mode.
<img width="587" alt="Screenshot 2024-09-16 at 3 04 04 AM" src="https://github.com/user-attachments/assets/a65525bf-66a4-45d7-9023-5de48be22ad1">

Reply:
Okay, fixing it for both light and dark modes and checking in all the major browsers

Reply:
Making the following changes fixes the color contrast issue as checked in all major browsers:

(But I couldn't find a way without using a `@media` query for color schemes, and I didn't wanted to re-color the states of anchor tag, which by default comes with accessibility out of the box.)

```css
.news {
  background-color: bisque;
  padding: 1em;
  border: solid thin black;

  @media (prefers-color-scheme: dark) {
    background-color: black;
  }
}

menu {
  list-style-type: none;
  display: flex;
  padding: 0;
  margin-bottom: 0;
  gap: 1em;
}

button {
  color: black;
  background: white;
}
```

![Screenshot 2024-09-16 at 6 14 23 PM](https://github.com/user-attachments/assets/c3df4a8c-769b-4568-b503-34eb19acfc3c)
![Screenshot 2024-09-16 at 6 14 29 PM](https://github.com/user-attachments/assets/fe7c4d19-7828-4ed1-a515-6a6af36d1852)
![Screenshot 2024-09-16 at 6 16 10 PM](https://github.com/user-attachments/assets/59bfb1bd-f895-4abd-9da8-ce9e38769d5a)
![Screenshot 2024-09-16 at 6 15 37 PM](https://github.com/user-attachments/assets/70b12565-d365-440d-91f7-1dbfab170f09)

@estelle Should I go for it?